cli/command/container/exec.go

```diff
@@ -16,24 +16,24 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type execOptions struct {
-	detachKeys  string
-	interactive bool
-	tty         bool
-	detach      bool
-	user        string
-	privileged  bool
-	env         opts.ListOpts
-	workdir     string
-	container   string
-	command     []string
-	envFile     opts.ListOpts
+type ExecOptions struct {
+	DetachKeys  string
+	Interactive bool
+	Tty         bool
```

Comment:
I guess the one on types.ExecConfig is generated from swagger, so we can't do the same?

Reply:
Hm... good question; actually, I don't think it is generated. Looking at https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/api/types/configs.go#L30-L44

I don't see a "generated" comment in that file, so that one was probably done manually (but using the wrong casing 😞). The files that were generated have a comment somewhere at the top; https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/api/types/plugin_env.go#L3-L4

```go
// This file was generated by the swagger tool.
// Editing this file might prove futile when you re-run the swagger generate command
```

But it's a good point; perhaps (as a follow-up?) we should either use that type for the options here and/or create an alias for it in the `api` (or `client`) package.